### PR TITLE
Refactor/enable linters

### PR DESCRIPTION
**Pull request by frosty-geek**
Since the code has grown "organically" over the years, it's very hard to review changes or add to the code base. 

This draft PR is intended to clean-up the code enough so that linters like 'bashate', 'cspell', 'language_server', 'shell', 'shellcheck' can actually assist to review changes.

ref: #118

### CODE REVIEW

**Comment by garloff** [comment]
Thanks, Ralf!
I'll review these carefully!

**Comment by garloff** [comment]
I have these rebased (cherry-picked them) in a new branch enable-linters2.
As your PR is marked "work in progress", I wonder whether you intended further work before merging, @frosty-geek ?
Let me know, so we can open the PR.

**Comment by garloff** [comment]
So I opened #127 (currently as draft).

**Comment by frosty-geek** [comment]
Hi Kurt,

> As your PR is marked "work in progress", I wonder whether you intended further work before merging, @frosty-geek ?

My intension was to work on this further but since it was such a huge undertaking that I marked this as WiP.

I'm with getting a this done whichever way you prefer.

Ralf

File: api_monitor.sh [hunk 1]
```diff
@@ -1082,48 +1276,65 @@ STATE=""
...
+    local MRNM=$5
+    local STIME=$6
+    local IDNM=$7
+    shift
+    shift
+    shift
+    shift
+    shift
+    shift
+    shift
```
Comment (garloff) [inline api_monitor.sh:1294]: `shift 7`
would save a lot of screen real estate.

**Comment by frosty-geek** [comment]
closing this as discussed
ref #127

**Pull request closed without merging**